papers/tesseroid-variable-density.md

```diff
@@ -1,20 +1,27 @@
 ---
 title: "Gravitational field calculation in spherical coordinates using variable densities in depth "
-date: 2019-07-01
+date: 2019-06-05
 author: santisoler, agustina, uieda, mario
 repository: pinga-lab/tesseroid-variable-density
 journal: Geophysical Journal International
+supplement: 10.6084/m9.figshare.8239622
+preprint: 10.31223/osf.io/3548g
+doi: 10.1093/gji/ggz277
+citation: "Soler, S. R., Pesce, A., Gimenez, M. E., & Uieda, L., 2019. Gravitational field calculation in spherical coordinates using variable densities in depth, Geophysical Journal International, doi:10.1093/gji/ggz277"
 related_papers: paper-tesseroids-2016
 alm: false
```

Comment:
@leouieda I don't know that `alm` stands for and what does it do. Feel free to set it to `true` if needed.

Reply:
I suspect that `alm` == "Article Level Metrics"... am I right?

Reply:
Yes!

Reply:
Done! :+1: